src/db/migrations/20230711034311_remove_temp_migration_col.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+export async function up(knex) {
+  if (true === await knex.schema.hasColumn('subscribers', 'db_migration_1')) {
+    await knex.schema.alterTable('subscribers', table => {
+      table.dropColumn('db_migration_1')
+    })
+  }
+
+  if (true === await knex.schema.hasColumn('subscribers', 'db_migration_2')) {
+    await knex.schema.alterTable('subscribers', table => {
+      table.dropColumn('db_migration_2')
+    })
+  }
+}
+
+export async function down(knex) {
+  if (true === await knex.schema.hasColumn('subscribers', 'db_migration_1')) {
+    console.log('do nothing for db_migration_2')
```

Comment:
```suggestion
    console.log('do nothing for db_migration_1')
```

Reply:
Actually, do we need anything in `down()`? These are just doing a column lookup on the `subscribers` table for column `db_migration_1` and `db_migration_2` and then doing a noop.